engine/glfw/java/com/dynamo/android/DefoldActivity.java

```diff
@@ -522,4 +523,39 @@ public String getGameControllerDeviceName(int deviceId) {
         }
         return name;
     }
+
+    /**
+     * Method to get meta-data value by key from AndroidManifest.xml
+     * @param key
+     * @return String
+     */
+    protected String getMetadata(String key) {
```

Comment:
Neat function.
Will this require any extra permissions?

Reply:
No, extra permissions are not required.